packages/web-runtime/src/components/ApplicationsMenu.vue

```diff
@@ -21,11 +21,22 @@
     >
       <ul class="uk-grid-small uk-text-center" uk-grid>
         <li v-for="(n, nid) in menuItems" :key="`apps-menu-${nid}`" class="uk-width-1-3">
-          <a v-if="n.url" key="apps-menu-external-link" :target="n.target" :href="n.url">
+          <a
+            v-if="n.url"
+            id="runtime-apps-menu-external-link"
```

Comment:
i think ids are the wrong thing to use here since its inside of an v-for, so you would have multiple a tags with the same id